main.js

```diff
@@ -0,0 +1,91 @@
+const toDoList = [ 
+	{name: 'погулять', status: 'In progress', priority: 'low'}, 
+    {name: 'сходить на рыбалку', status: 'Done', priority: 'high'},
+    {name: 'почитать', status: 'In progress', priority: 'high'}
+];
+
+const STATUSES = [ "To Do", "In progress", "Done" ]
+const PRIORITY = [ 'low', 'high' ]
+
+function checkName(name) {
```

Comment:
Это не критично, но я бы назвал например:
```suggestion
function isTaskExists(name) {
```